packages/ui/src/components/organisms/DynamicForm/DynamicForm.tsx

```diff
@@ -20,61 +27,79 @@ interface Props<TFormData> {
   formData?: object;
   warnings?: InputsWarnings;
   disabled?: boolean;
+  fields?: Record<keyof RegistryFieldsType, RJSVInputAdapter<any>>;
+  layouts?: Partial<TemplatesType>;
+  liveValidate?: boolean;
   transformErrors?: (errors: RJSFValidationError[], uiSchema: UiSchema) => RJSFValidationError[];
   onChange?: (formData: TFormData) => void;
   onSubmit: (formData: TFormData) => void;
 }
 
-export function DynamicForm<TFormData extends object>({
-  schema,
-  uiSchema,
-  className,
-  formData,
-  warnings,
-  disabled,
-  transformErrors,
-  onChange,
-  onSubmit,
-}: Props<TFormData>) {
-  const handleChange = useCallback(
-    (form: IChangeEvent<TFormData>) => {
-      onChange && onChange(form.formData);
-    },
-    [onChange],
-  );
+export const DynamicForm = forwardRef(
+  <TFormData extends object>(
```

Comment:
Prefer `Record<PropertyKey, unknown>` over `object` as its usually the intended type. 🙂